spec/services/nomis/api_spec.rb

```diff
@@ -43,22 +43,43 @@
     let(:params) {
       {
         offender_id: 1_055_827,
-        start_date: Date.parse('2016-04-01'),
-        end_date: Date.parse('2016-04-21')
+        start_date: Date.parse('2016-05-01'),
+        end_date: Date.parse('2016-05-21')
```

Comment:
Yes, which we can't avoid because the NOMIS API is a black box from the perspective of these tests. It shouldn't be an issue because we won't need to re-record often.